classes/Upsell.php

```
@@ -128,68 +128,409 @@ private static function detect_integrations() {
+	 */
+	private static function get_notice_bar_triggers() {
+		static $triggers;
+
+		if ( isset( $triggers ) ) {
+			return $triggers;
+		}
+
+		$license_service = \PopupMaker\plugin( 'license' );
+		$license_tier    = $license_service->get_license_tier();
+		$license_status  = $license_service->get_license_status();
+		$pro_is_active   = \PopupMaker\plugin()->is_pro_active();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$pro_is_active'. (undefined)

(UnusedLocalVariable)